common/src/main/kotlin/streams/config/StreamsConfigExtensionFactory.kt

```diff
@@ -15,6 +16,6 @@ class StreamsConfigExtensionFactory: ExtensionFactory<StreamsConfigExtensionFact
     }
 
     override fun newInstance(context: ExtensionContext, dependencies: Dependencies): Lifecycle {
-        return StreamsConfig(dependencies.log())
+        return StreamsConfig(dependencies.log(), dependencies.dbms())
```

Comment:
I guess due to API changes this sort of thing needs to get backported to 3.5 because I think this API is a bit different right?

Reply:
Sorry, what do mean? This version is specific for Neo4j 4.0. We'll continue to maintain the version for Neo4j 3.5 in his branch.

Reply:
Nevermind -- I got confused, this comment doesn't matter, you're right.